frontend/elements/src/entry/home/sections/header-section.ts

```diff
@@ -0,0 +1,94 @@
+ import "@elements/entry/home/TOSORT/column-list";
+ import "@elements/core/buttons/rectangle";
+ 
+
+import { LitElement, html, css, customElement, property } from 'lit-element';
+@customElement('header-section')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+
+   main{
+    // width: 88px;
+    height: 512px;
+    display:flex;
+   }
+
+.logo{
+  width: 115.4px;
```

Comment:
the height and width here is unneccessary. you should set the width and height should be auto.